frontend/javascripts/main.tsx

```diff
@@ -27,8 +27,9 @@ import Model from "oxalis/model";
 import { setupApi } from "oxalis/api/internal_api";
 import { setActiveOrganizationAction } from "oxalis/model/actions/organization_actions";
 import checkBrowserFeatures from "libs/browser_feature_check";
-// @ts-ignore
-import _olvy from "libs/olvy.js";
+
+// Suppress warning emitted by Olvy because it tries to eagerly initialize
+window.OlvyConfig = null;
```

Comment:
Note to myself: Double-check that this is working. Last time I tested, I saw the warning, anyways.

Reply:
After checking, it doesn't seem to be needed any longer. The warning I saw was "[Olvy] - Target element not found. Call Olvy.show() to open your releases widget manually", but that is printed on master, too. The cause is that we are using a dummy target element for olvy. Relevant comment

```javascript
// This target needs to be defined (otherwise, Olvy crashes when using .show()). However,
// we don't want Olvy to add any notification icons, since we do this on our own. Therefore,
// provide a dummy value here.
target: "#unused-olvy-target",
```